geom/src/quadratic_bezier.rs

```diff
@@ -572,7 +597,7 @@ fn monotonic_solve_t_for_x() {
     for i in 0..10u32 {
         let t = i as f32 / 10.0;
         let p = curve.sample(t);
-        let t2 = curve.assume_monotonic().solve_t_for_x(p.x, 0.0..1.0, tolerance);
```

Comment:
I don't understand what happened to the tolerance here 😅

Reply:
The monotonic quadratic bézier implementation of `solve_t_for_x` doesn't need a tolerance (nor a narrowed range of t). The generic monotonic implementation of that function used with cubic curves does, though.

Reply:
Oh that's why, thank's a lot for your explanation :)